.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "julia.environmentPath": "/home/radu/NeuralAttentionlib.jl"
+}
```

Comment:
This file should be removed

Reply:
Yes. sorry my bad can you test it before merging? I didn't test it that much.

Reply:
I don't have any device with amdgpu, so I can only test if the code compiled without issues.